<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>333</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>473</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>473</width>
    <height>333</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Checkbox</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>55</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="cxVerif1">
       <property name="text">
        <string>Caixa 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cxVerif2">
       <property name="text">
        <string>Caixa 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cxVerif3">
       <property name="text">
        <string>Caixa 3</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnVer1">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Verificar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnVer2">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Vetor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnVer3">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Estado</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
